tests/test_scheduler.py

```diff
@@ -0,0 +1,47 @@
+# pylint: disable=all
+import pytest
+
+from src.uwtools.scheduler import JobScheduler
```

Comment:
Whats `src`?
This should be `uwtools.scheduler`.

Reply:
I believe that is why the pytest is failing.

Reply:
@aerorahul thanks for pointing this out; needed to make an update to the github action.

Reply:
I don't think this is correct. The install should not be src.uwtools

Reply:
It works both ways, as we're specifying to look undre 'src' in 'setup.cfg'.

Removing the prefix 'src'.